addons/pos_stripe/static/src/js/payment_stripe.js

```diff
@@ -57,6 +53,17 @@ let PaymentStripe = PaymentInterface.extend({
     },
 
     checkReader: async function () {
+        try {
+            if ( !this.terminal ) {
+                let createStripeTerminal = await this.createStripeTerminal();
```

Comment:
No need to await here. as mention below, `createStripeTerminal` doesn't need to be `async`